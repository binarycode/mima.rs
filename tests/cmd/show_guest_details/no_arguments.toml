args = 'show-guest-details'
status.code = 2
