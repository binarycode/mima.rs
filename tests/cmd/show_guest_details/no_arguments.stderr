error: The following required arguments were not provided:
    <GUEST_ID>

USAGE:
    mima show-guest-details <GUEST_ID>

For more information try --help
